test/fw/ptl/utils/pbs_procutils.py

```diff
@@ -282,12 +281,19 @@ def set_frequency(self, value=60):
         self.logger.debug('procmonitor: set frequency to ' + str(value))
         self.frequency = value
 
-    def get_system_stats(self, nw_protocols=['TCP']):
+    def get_system_stats(self, nw_protocols=sentinel):
         """
         Run system monitoring
         """
+        if nw_protocols is sentinel:
+            self.skipTest("network protocol values is not set.")
```

Comment:
@vstumpf used the sentinel instead of default.

Reply:
Sorry, i meant use something like
```Python3
def get_stats(self, mw_proto=None):
    # if no protocols set, use default 
    if not nw_proto:
        nw_proto = ['TCP']
```
If you think you should skip the test, that also works.
If you're curious, take a look at [this link](https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html) to see why I made the original comment.